CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+This template and our [recommendation pages][website] were made by [RSE]s at
+[UCL's Centre for Advanced Research Computing][UCL ARC]. We made it with
+Research Software projects in mind, but whoever you are, we hope you find this
+useful!
+
+We are actively encouraging users to ask questions and start discussions in the
+[discussions tab] of this repository. Does something seem like it's broken?
+Please go ahead and [open an issue]!
+
+The [website] pages are open to contributions but they will need to be reviewed
+by a member or associate member of ARC. We might be slow to approve new tool
+suggestions (since we'll probably want to discuss them first) but don't let that
+put you off creating an issue.
+
+## Development workflow
+
+To contribute a change, please:
+
+1. Create a new branch (or fork if you're not in ARC-collaborations).
+2. Modify an existing page, create a new one, or tweak the template.
+   a. Run [pre-commit](https://pre-commit.com) which will lint your changes.
+   b. Check the tests pass if you modified the template (`pytest -s`).
+3. Open a pull request with changes.
+4. Ask someone who did not contribute to the PR from ARC-collaborations to
+   review it.
+5. If approved with no comments, then the reviewer will merge the PR.
+6. If changes are requested, the PR author should address the comments, and then
+   ask for review again.
+
+<!-- links here -->
+
+<!-- prettier-ignore-start -->
+[website]: https://github-pages.arc.ucl.ac.uk/python-tooling
+[UCL ARC]: https://ucl.ac.uk/arc
+[open an issue]: https://github.com/UCL-ARC/python-tooling/issues/new/choose
+[Discussions tab]: https://github.com/UCL-ARC/python-tooling/discussions
+[RSE]: https://society-rse.org/about/history
```

Comment:
Curious as to why this syntax? Always makes me think of RST

Reply:
I like it, but literallyonly because it doesn't interrupt the flow of the text if you read the raw markdown.

So I can write...
```
We are actively encouraging users to ask questions and start discussions in the
[discussions tab] of this repository. Does something seem like it's broken?
Please go ahead and [open an issue]!
```

Reply:
For example: I would often read raw markdown by `cat` ing things in the terminal.

Reply:
Yeah I like that bit, less a fan of [][]